### DISCUSSION TITLE
Translation to Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by goliv04053** [jocmp/capyreader#239] [Feature Requests]
I would like to help but how can I do it? @jocmp

**Comment by jocmp** [comment]
@goliv04053 thanks for your help. I readded [Brazilian Portuguese](https://hosted.weblate.org/translate/capy-reader/strings/pt_BR) to the Weblate project. I don't know why it showed zero translation at first. You should be able to add translations now.
